tests/cargo-kani/output-format/main.expected

```diff
@@ -0,0 +1,2 @@
+Description: "assertion failed: 1 + 1 == 2"
+Description: "dead object in OBJECT_SIZE(&temp_0)"
```

Comment:
Are we making the new output the default output for Cargo Kani?

Reply:
No, this test uses `--output-format regular`:
```
[package.metadata.kani]
flags = {output-format = "regular"}
```

Reply:
Thanks, maybe it's time to make it the default output if so much new work depends on it.